### ISSUE TITLE
All pending tests are displayed in "Without feature" category

### ISSUE THREAD

**Issue by psprogis** [allure-framework/allure-jasmine#17]
We have lots of tests marked by ```allure.feature``` to assign each test to specific feature, e.g. (small part of xUnit widget):
![2017-02-07_1133](https://cloud.githubusercontent.com/assets/7395225/22685553/696391f2-ed29-11e6-9339-9efc3cacd827.png)

It looks correct, each spec contains its our passed, failed, pending tests, etc.

But in Features widget and Behaviors tab all pending test goes to "Without feature"
![2017-02-07_1138](https://cloud.githubusercontent.com/assets/7395225/22685761/42daaa1a-ed2a-11e6-9ced-bc4bb487005f.png)

So we cannot see "pending test per feature".

Steps to reproduce: try the following example and generate report
```javascript
describe('my suite', () => {
    it('foo feature should do something', () => {
        allure.feature('foo feature');
        expect(true).toBe(true);
    });

    it('foo feature is good', () => {      
        allure.feature('foo feature');
        expect(true).toBe(true);
    }).pend('issue 8435');

    xit('bar feature', () => {
        allure.feature('bar feature');
        expect(1).toBeTruthy();
    });
});
```

Expected result: 2 features, foo: 1 passed test, 1 pending test; bar: 1 pending test
![1](https://cloud.githubusercontent.com/assets/7395225/22686092/7e202694-ed2b-11e6-971f-c982d98f84eb.png)

Actual result: 1 feature, foo: 1 passed test, Without feature: 2 pending tests
![2](https://cloud.githubusercontent.com/assets/7395225/22686167/b867c7e4-ed2b-11e6-9a3b-5fcd2c123799.png)

**Comment by deepakaravind** [comment]
Agreed with @psprogis.  Am running into a similair issue - in our case, we are forcefully xit the test specs so that they end up in pending state for that specific allure.feature.  Is there a solution for this?  Any help/response is appreciated.

**Comment by just-boris** [comment]
The current Allure API accepts additional metadata during test run time. Pending tests are not run at all, therefore all their content, that includes `allure.feature()` and `allure.story()` unavailable.

In order to fix the issue, we need to consider another way to add features and stories. So far I see two possible options

1. Enhance `it()` method signature. We may make it accept something like `it("test name", {story: "story name"}, () => {/*code*/})`. But I will break the initial Jasmine API, that we would like to avoid
2. Encode story and feature tags into test name: `it("test name [story=story name]", () => {/**/})`. The text within brackets will be extracted and parsed as feature and story. 

Both options have their own benefits and downsides and both will be a breaking change, so I'd like to ask for your feedback first. What do you think about them. Do you have your own third option?

**Comment by psprogis** [comment]
both options look so-so... I guess second one is better, at least we will not break it signature and it just an additional info in method name and it is needed only for some pending tests.
